<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountInfo</class>
 <widget class="QDialog" name="AccountInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="accountIdLabel">
       <property name="text">
        <string>Account ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="accountIdLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="userNameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="oFXFidLabel">
       <property name="text">
        <string>OFX fid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="oFXFidLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="oFXFiorgLabel">
       <property name="text">
        <string>OFX fiorg</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="oFXFiorgLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="oFXBankIDLabel">
       <property name="text">
        <string>OFX Bank ID</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="oFXBankIDLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="oFXURLLabel">
       <property name="text">
        <string>OFX URL</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="oFXURLLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="oFXAccountTypeLabel">
       <property name="text">
        <string>OFX Account Type</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="oFXAccountTypeComboBox">
       <item>
        <property name="text">
         <string>Credit Card</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Checking</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Savings</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Money Market</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Credit Line</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="scraperClassLabel">
       <property name="text">
        <string>scraperClass</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="scraperClassLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>45</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
